recipes-core/rcu-service/rcu-service_.bb

```diff
@@ -0,0 +1,29 @@
+SUMMARY = "RCU Service"
```

Comment:
come to think of it, seems like we need include the version in the recipe filename, else we will have no way to know what is version of "rcu-service" is currently installed into the image

Reply:
Added version number 1.0 to the recipe for now.